src/app/events/org-events.component.ts

```diff
@@ -75,11 +75,16 @@ export class OrgEventsComponent implements OnInit {
     })
   }
 
+  public taskSearchTabName() {
+    return this.isPublicView() ? 'חיפוש הצעות נסיעה' : 'חיפוש נסיעה'
+
+  }
+
   private loadEvents() {
     let date = new Date()
     date.setHours(date.getHours() - 2)
     let orgFilter: EntityFilter<Task> = {}
-    if (remult.user!.showAllOrgs == null) {
+    if (remult.user?.showAllOrgs == null) { // TODO are volunteers allowed to see all orgs?
```

Comment:
They can only see tasks of orgs that they are part of - see api prefilter on tasks